webpack.config.js

```diff
@@ -48,6 +48,10 @@ module.exports = (shell = {}) => {
           'react-dom$': 'react-dom/profiling',
           'scheduler/tracing': 'scheduler/tracing-profiling',
         }),
+        '@bbc/gel-foundations': path.resolve(
```

Comment:
This will remove the duplicate bundles, but feels a little dangerous as different packages rely on different major versions! Is it possible with our setup to publish a new patch for the previous major version to update this dependency? Or would we need to create a whole new package?

Reply:
Update: we've decided to temporarily ignore gel-foundations in the dependency check. This ignore will be removed in a future issue: https://github.com/bbc/simorgh/issues/5625